pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -378,6 +508,15 @@ func AWSAccountIDClaimed(accountClaim *awsv1alpha1.AccountClaim, accountClaimLis
 	return match
 }
 
+func checkOUMapping(cMap *corev1.ConfigMap) (string, string, error) {
+	baseID := cMap.Data["base"]
```

Comment:
These should be wrapped in an Ok block in case they don't exist.

Reply:
I did some manual testing and it did not fail but rather returned an empty string when the values did not exists. That is why I make [this](https://github.com/openshift/aws-account-operator/blob/c274fabb2ac1df6c1546882e570fc71f279c3916/pkg/controller/accountclaim/accountclaim_controller.go#L514-L516) check. If you, however, think it's better in the Ok block I do not have strong feelings about it.

Reply:
The values or the keys?

Reply:
the keys, sorry

Reply:
If you wrap both in ok blocks, that elimates the need for 436 and is semantically the "golang" way. Curious about other's opinions though